snowpack/src/commands/build.ts

```diff
@@ -125,7 +125,13 @@ class FileBuilder {
           }
 
           case '.html': {
-            code = wrapHtmlResponse({code, isDev: false, hmr: false, config: this.config});
+            code = wrapHtmlResponse({
+              code,
+              isDev: false,
+              hmr: false,
+              config: this.config,
+              mode: 'production',
+            });
```

Comment:
I had to add the `mode` option to differentiate between the build & dev commands